binary_transparency/firmware/cmd/ft_personality/internal/http/server_test.go

```diff
@@ -162,7 +178,13 @@ func TestAddFirmware(t *testing.T) {
 				t.Errorf("error response: %v", err)
 			}
 			if got, want := resp.StatusCode, test.wantStatus; got != want {
-				t.Errorf("status code got != want (%d, %d)", got, want)
+				var desc string
+				body, err := ioutil.ReadAll(resp.Body)
+				if err == nil {
+					// Don't error if we don't get the body, that's not the goal of this test
+					desc = string(body)
+				}
```

Comment:
What does this bit do? (the comment seems to suggest that it's not really important for the test?)

If a test-case errors then we'll see the `test.desc` logged (e.g. `TestAddFirmware/valid test but trillian failure. FAIL") so we can look in here and see what the body was from the testcase.  But perhaps there's a different intent that I've not understood.

Reply:
Added originally by Martin, to debug I guess, so removed it as I feel it is not relevant.